assets/queries/cloudFormation/s3_bucket_should_have_bucket_policy/test/negative.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A sample template
+Resources:
+  S3Bucket:
+    Type: 'AWS::S3::Bucket'
+    DeletionPolicy: Retain
+    Properties:
+      BucketName: DOC-EXAMPLE-BUCKET
+  SampleBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: DOC-EXAMPLE-BUCKET
```

Comment:
Bucket name should be lowercase. Must fit this regex '^[a-z0-9][a-z0-9.-]*[a-z0-9]$'